task-sdk/src/airflow/sdk/definitions/asset/decorators.py

```diff
@@ -98,6 +99,18 @@ def determine_kwargs(self, context: Mapping[str, Any]) -> Mapping[str, Any]:
         return dict(self._iter_kwargs(context))
 
 
+def _instanciate_task(definition: AssetDefinition | MultiAssetDefinition) -> _AssetMainOperator | XComArg:
```

Comment:
I believe the correct spelling is _instantiate_ instead.

Reply:
Also we probably do not need to return anything from this function?

Reply:
Good catch! I fixed the misspelling.

I also removed the unnecessary `return` and it works the same as before.